content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -167,6 +167,39 @@ fail validation.
 				</ul>
 			</td>
 		</tr>
+		<tr>
+			<td style="white-space: nowrap">Probes or Lifecycle Hooks with Host field</td>
+			<td>
+				<p>HostProbeOrHostLifecycle should be disallowed entirely</p>
```

Comment:
```suggestion
				<p>The Host field in probes and lifecycle hooks must be disallowed.</p>
```

Reply:
ah oops didn't see this one...good comment too bad I didn't see it with the other ones :( cause it only popped up when I refreshed.
but I can fix this in a FUP because I just see @liggitt lgtm'ed the PR already and today is the freeze.
note-to-self: not to write a long word with multiple words in it,makes no sense